numba/core/event.py

```diff
@@ -423,3 +426,42 @@ def on_exit(*exc_details):
 
         start_event(kind, data=data)
         yield
+
+
+def _write_chrome_trace(rec):
+    """ If chrome tracing is enabled,
+        write event information (process/thread id, event name,
+        and duration time to json file.
```

Comment:
This doc string is outdated for the contents of this function? There's no write or json involved, looks like it's just parsing event records into a list of dicts for consumption by the json dumping function later?